build/build.sh

```diff
@@ -27,7 +33,7 @@ mkdir -p ${BUILD_DIR}
 pushd ${BUILD_DIR}
 curl -sL ${URL} | tar Jxf - --strip-components=1
 # https://stackoverflow.com/questions/37060747/escaping-origin-for-libtool-based-project
-./configure LDFLAGS="-Wl,-rpath,'\$\$ORIGIN/../lib'" --without-curses --with-math=gmp --with-db --with-xml2 --with-json-json-c
+./configure LDFLAGS="-Wl,-rpath,'\$\$ORIGIN/../lib'" --with-math=gmp --without-db --without-curses --without-xml2 --without-json
```

Comment:
TODO: add target directory (where GnuCOBOL is unpacked to later), if know via `--prefix` parameter

Reply:
We can force a fixed prefix but that will possibly break any other installation. I'm ok with it though if you are